docs/3.x/src/guide/veevalidate.md

```diff
@@ -403,3 +403,38 @@ The form-level `validation` object contains the following properties:
   </SchemaForm>
 </template>
 ```
+
+The following are a few common examples for behaviors implemented with the form-level validation state.
+
+**Disable buttons until all fields are valid**
+
+```html
+<SchemaForm :schema="schema">
+  <template #afterForm="{ validation }">
+    <button :disabled="!validation.meta.valid">Submit</button>
+  </template>
+</SchemaForm>
+```
```

Comment:
Yeah there's some a11y considerations here, I don't want FVL docs to become a11y docs, but I also don't want to promote bad practices. We can keep the example if:

- Let's switch `:disabled` for `:aria-disabled`
- Add a subnote `Learn more about accessible disabling of form buttons` and link it to: https://css-tricks.com/making-disabled-buttons-more-inclusive/